examples/Example1.cpp

```diff
@@ -5,16 +5,26 @@
 #include <Arduino.h>
 #include <ABBAurora.h>
 
+
+
 #define RX2 16
 #define TX2 17
+#define RX1 3
+#define TX1 1
 #define INVERTER_ADDRESS 2
 #define TX_CONTROL_GPIO 4
+#define TX_CONTROL_GPIO2 16
 
 ABBAurora *inverter;
 void setup()
 {
     Serial.begin(115200);
+    
+#if defined(ESP832)
     ABBAurora::setup(Serial2, RX2, TX2, TX_CONTROL_GPIO);
+#elif defined(ESP8266)
```

Comment:
here we have a huge issue: 
with this new code, there is no support to Arduino anymore. Just ESp32 and ESp8266 but not for Atmel micro-controllers.